### ISSUE TITLE
Missing information in inception_train.py

### ISSUE THREAD

**Issue by chrisrn** [tensorflow/tensorflow#5352]
I wanted to fine-tune inception network using my dataset, which is way too big. Converting to TFRecords format is not the best option because it uses a lot of GB in hard disk. I used a placeholder for images and a placeholder for labels and here https://github.com/tensorflow/models/blob/master/inception/inception/inception_train.py
I converted the command in line 336 in    _, loss_value = sess.run([train_op, loss], feed_dict={images_placeholder: images , labels_placeholder: labels}). I think I achieved fine-tuning, so you could add it to the documentation.

**Comment by aselle** [comment]
It's difficult to follow exactly what you are requesting and what you did. We'd appreciate any improvements to the documentation based on your experience as a pull request. Thank you!

**Comment by aselle** [comment]
Automatically closing due to lack of recent activity, we will reopen if further information becomes available. Thanks!
